tests/client/test_machine.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import tinychain as tc
+import unittest
+from sklearn.datasets import load_breast_cancer
```

Comment:
I think it would be easier to debug using a small dataset, e.g. inputs `[0, 1, 2, 3]` and labels `[1, 3, 5, 7]` to check that regression correctly fits `y = 2x + 1`